packages/paste-website/src/pages/layout/stack/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Stack - Layout
+description: Stack is a horizontal and vertical flow component used to build layouts.
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Card} from '@twilio-paste/card';
+import {FormInput, FormLabel} from '@twilio-paste/form';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Stack} from '@twilio-paste/stack';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/stack/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteLayout(filter: {name: {eq: "@twilio-paste/stack"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/stack/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Stack"
+  categoryRoute={SidebarCategoryRoutes.LAYOUT}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/layout/stack"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/layout-stack--stack"
+  data={props.data.allPasteLayout.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Stack
+
+Stack is a layout component that allows you to easily stack elements together with a space between them.
```

Comment:
Remove Stack is a layout component after moving the line above here. Change to "It allows you to"

Reply:
change "with a space" to "with defined spacing between them"

Reply:
Reduce occurrences of "Stack" since it has been established in the leading paragraph. You can say "It can be set up either vertically or horizontally"

Reply:
Done.